code/game/area/areas/shuttles.dm

```diff
@@ -67,6 +67,7 @@
 /area/shuttle/transit
 	name = "Hyperspace"
 	desc = "Weeeeee"
+	dynamic_lighting = DYNAMIC_LIGHTING_DISABLED
```

Comment:
Reasoning?

Reply:
@SpaceManiac I don't think lighting ever worked in transit space?

Reply:
It's being disabled on the turf level AFAIK, I doubt this will be a problem, was just curious what prompted it.